design/tlog-spilling.md.html

```diff
@@ -262,8 +262,8 @@
 A TLog is all the data that is private to one generation.  Most notably, the
 1.5GB mutation buffer and the on-disk files are owned by the `SharedTLog`.  The
 index for the data added to that buffer is maintained within each TLog.  In the
-code, a SharedTLog is `struct TLogData`, and a TLog is `struct LogData`.
-(I didn't choose these names.)
+code, a SharedTLog is called `struct TLogData`, and a TLog is `struct LogData`.
+(These names can be confusing at first).
```

Comment:
I changed the sentence here because I think this is rather what was meant indirectly.

After a while, I think the naming starts to make some sense if I understand it correctly. TLogData is the data of the TLogServer actor itself, and each instance of a generation of TLog is LogData (which can be thought as the last one is the current log for the new commits), and each of them has TagData(s) for Storage Servers they serve.